ecf/dev/sub_level2.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -login
```

Comment:
why does one need a script to do:

cd directory_name
qsub script_name

Reply:
the script is deleted